runtime/src/lib.rs

```diff
@@ -652,12 +626,8 @@ parameter_types! {
 	pub const ExecutiveBody: BodyId = BodyId::Executive;
 }
 
-// We allow root and the Relay Chain council to execute privileged collator selection operations.
-pub type CollatorSelectionUpdateOrigin = EnsureOneOf<
-	AccountId,
-	EnsureRoot<AccountId>,
-	EnsureXcm<IsMajorityOfBody<RocLocation, ExecutiveBody>>,
->;
+// We allow root only to execute privileged collator selection operations.
+pub type CollatorSelectionUpdateOrigin = EnsureRoot<AccountId>;
```

Comment:
Thanks for catching this, it was unintended.

Reply:
So the configuration here was the same as in the `parachain-template` and there this change was introduced (specifically [here](https://github.com/paritytech/cumulus/commit/496fab27b5631fb95383f48586cd593ae3f32a7d#diff-447906ba563849a582953cc1753f58164c23d4902539e37c5dde387113a3aa5bL590-R568)). `EnsureOneOf` has also been removed from Substrate and is no longer available.

Maybe @NukeManDan or @riusricardo can clarify if applying this change for `canvas` is correct?

Reply:
So this change comes from @apopiak (see comment [here](https://github.com/substrate-developer-hub/substrate-parachain-template/pull/80#discussion_r738197404)). In our case, since we're planning to be a CGP I think we should leave it as it was before

Reply:
As stated, this is not possible `EnsureOneOf` has been removed.

Reply:
yep